### ISSUE TITLE
Add new excavator build ?

### ISSUE THREAD

**Issue by ramius86** [DillonN/NiceHashMiner#23]
https://github.com/nicehash/excavator/releases

Seems got very nice hashrate

**Comment by CreamyG31337** [comment]
As @DillonN pointed out in #20 ,
"WARNING!!! Excavator is a proprietary software by NiceHash and has a special EULA. YOU ARE NOT ALLOWED TO REDISTRIBUTE IT!"
We might be able to get permission or just add support and you'll have to download and extract the zip file yourself. It's only a tiny bit faster than Dual Mining on my 1060, so you're not missing much anyways.

**Comment by DillonN** [comment]
Yes for the time being at least this won't be added. Will have an update on the status next week

**Comment by ramius86** [comment]
Mh I see but can't simply download excavator from their source like other miners without redistribuite it and only add config parameters?

**Comment by pducharme** [comment]
What about the Claymore Dual Mining 9.6 ?  You have 9.4 in there?

**Comment by DillonN** [comment]
@pducharme CD is at 9.5 currently, there are no speed benefits to 9.6 as far as I can tell

**Comment by pducharme** [comment]
Ok, good then, no need to update? just want to get +15% on my 1060 that I saw last week.
